### PR TITLE
added sso content and updated content-nav

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: docs/modules/ROOT/pages/browser-sso.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+[source, properties]
+----
+dbms.jvm.additional=-Dorg.apache.commons.logging.Log=org.apache.commons.logging.impl.SimpleLog
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.showdatetime=true
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.log.org.apache.http=DEBUG
+----
+
+Browser needs to be aware of the identity providers available for use.
+This is done by specifying a URL parameter discoveryURL that specifies a URL to a json file containing the SSO providers.
+Example for Browser: `http://<browser-server-host>:<http-port>?discoveryURL=http://webhost.com/public/discovery.json`.
```
Comment (jharris4) [inline docs/modules/ROOT/pages/browser-sso.adoc:49]: @OskarDamkjaer can you please confirm this example url is correct?

**Comment by OskarDamkjaer** [comment]
I think it would make sense to mention that the `:debug` command also contains logging from our SSO implementation and is helpful for debugging

File: docs/modules/ROOT/pages/browser-sso.adoc (hunk 1 above)
Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/browser-sso.adoc:49]: Looks like correct example to me!

Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/browser-sso.adoc:49]: We could mention that it's planned to be picked up from the server in the future

File: docs/modules/ROOT/pages/browser-sso.adoc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+// It contains all the possible parameters you can provide.
+// You most likely do not need all the parameters.
+// If you are unsure, please consult your Neo4j representative to avoid misconfiguration.
+//
+// [source, parameters]
+// ----
+//
+// {
+// 	// other discovery entries
+// 	// e.g. "bolt": "bolt://localhost:7687"
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/browser-sso.adoc:62]: While not strictly required, I think it's a good idea to have the "bolt" field included in your discovery data. If it's not present here, in the built-in discovery or in the connectURL param we don't know where to login to when we come back from the SSO server

Comment (AlexicaWright) [reply docs/modules/ROOT/pages/browser-sso.adoc:62]: This section is about the `discovery.json` file and that doesn't apply to Browser, if I'm not mistaken? Where should I include it?

File: docs/modules/ROOT/pages/browser-sso.adoc (hunk 1 above)
Comment (AlexicaWright) [reply docs/modules/ROOT/pages/browser-sso.adoc:49]: What is planned to be picked up from the server?

Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/browser-sso.adoc:49]: There's a built in discovery.json as well when the browser is hosted by neo4j, it currently only has `bolt` and some other fields but in the future it'll be configurable to have SSOProviders as well. But we can wait with documenting that until it's available in 4.4

File: docs/modules/ROOT/pages/browser-sso.adoc (hunk 2 above)
Comment (OskarDamkjaer) [reply docs/modules/ROOT/pages/browser-sso.adoc:62]: Ah sorry, yes these do apply to us as well!

File: docs/modules/ROOT/pages/browser-sso.adoc [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.showdatetime=true
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.log.org.apache.http=DEBUG
+----
+
+Browser needs to be aware of the identity providers available for use.
+This is done by specifying a URL parameter discoveryURL that specifies a URL to a json file containing the SSO providers.
+Example for Browser: `http://<browser-server-host>:<http-port>?discoveryURL=http://webhost.com/public/discovery.json`.
+
+// The `discovery.json` file must contain entries tailored to your organization’s specific SSO solution.
+// Below is a reference discovery file for the ID provider (IDP) _Keycloak_ containing one SSO provider.
```
Comment (OskarDamkjaer) [inline docs/modules/ROOT/pages/browser-sso.adoc:52]: Set up to be running on port "18080", it'd be worth to add here I think

**Review by OskarDamkjaer — APPROVED** [review]
Looks good, thanks!

**Pull request merged**